website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md

```diff
@@ -4,7 +4,7 @@ title: "Getting Started with Karpenter"
 linkTitle: "Getting Started with Karpenter"
```

Comment:
I think a docs file rename might be in order here too? From `preview/getting-started/getting-started-with-eksctl` to  `preview/getting-started/getting-started-with-karpenter`

Reply:
Yeah, sure. I think it's a leftover from when we had Terraform version here.

Reply:
Heads-up; it's going to be more changes updating it across the repo